dashboards/query-stats.json

```diff
@@ -1950,6 +1950,16 @@
         ],
         "query": "*",
         "type": "textbox"
+      },
+      {
+        "baseFilters": [],
+        "datasource": {
+          "type": "prometheus",
+          "uid": "$ds"
+        },
+        "filters": [],
+        "name": "adhoc",
+        "type": "adhoc"
```

Comment:
query stats dashboard uses VictoriaLogs as datasource. Prometheus-typed ad-hoc query won't make effect.

Reply:
fixed it and checked